src/sdk/namespace_fs.js

```diff
@@ -864,24 +884,25 @@ class NamespaceFS {
             params.obj_id
         );
     }
+
     async _load_multipart(params, fs_account_config) {
         await this._load_bucket(params, fs_account_config);
         params.mpu_path = this._mpu_path(params);
         try {
-            await nb_native().fs.stat(fs_account_config, params.mpu_path);
+            await _stat_with_permissions_check(fs_account_config, params.mpu_path);
         } catch (err) {
             // TOOD: Error handling
             if (err.code === 'ENOENT') err.rpc_code = 'NO_SUCH_UPLOAD';
             throw err;
         }
     }
 
-    async _create_path(dir, fs_account_config) {
+    async _create_path(dir, fs_account_config, mode) {
         let dir_path = path.isAbsolute(dir) ? path.sep : '';
         for (const item of dir.split(path.sep)) {
             dir_path = path.join(dir_path, item);
             try {
-                await nb_native().fs.mkdir(fs_account_config, dir_path, 0o777);
+                await nb_native().fs.mkdir(fs_account_config, dir_path, mode || 0o777);
```

Comment:
Why are we setting the mode? Why not let the default apply? In linux every user has a umask that sets the default mode for new files/dirs, so I would say that we want to use that as much as possible. If we can't then I would at least set the default in config like this:
```javascript
config.NSFS_DEFAULT_MODE_DIR=0o755;
config.NSFS_DEFAULT_MODE_FILE=0o644;
```

Reply:
see https://geek-university.com/linux/set-the-default-permissions-for-newly-created-files/

Reply:
if we need more granular control on the umask for each account we can add it to our nsfs_account_config...

Reply:
according to our discussion, umask (007) now applied in a global context. the only issue here is that files get -rw-rw-- permissions, I can change it by setting the mode to 667 but I find it too hardcoded. 
WDYT? @guymguym 
do we have to preserve the execute permissions on files?